api/conf/conf.go

```diff
@@ -79,12 +82,19 @@ type Config struct {
 }
 
 func init() {
-	flag.StringVar(&confDir, "c", "./conf/", "conf dir")
-	flag.Parse()
+	//go test
+	if strings.HasSuffix(os.Args[0], ".test") {
+		_, basePath, _, _ := runtime.Caller(0)
+		confDir = filepath.Dir(basePath) + "/"
+		fmt.Println("confDir:", confDir)
+	} else {
+		flag.StringVar(&confDir, "c", "./conf/", "conf dir")
```

Comment:
`flag.StringVar(&confDir, "c", "./conf/", "conf dir")`
to
`flag.StringVar(&confDir, "c", "conf/config.yaml", "config file path")`

Reply:
as we talked right now, we should use `flag.StringVar(&workDir, "p", ".", "current work dir")`.

support to specify the prefix path via ENV, `APISIX_API_WORKDIR` seems a good name for us.

we should support both of them.

```sh
# style 1
manager-api -p /home/xxx/apisix/manager-api

# style 2
APISIX_API_WORKDIR=/home/xxx/apisix/manager-api manager-api
```

Reply:
I would prefer style 1 IMO

Reply:
I would prefer style 2 IMO

Reply:
it's better. and user don't need to config other dirs or file paths.

Reply:
OK!